yarn-audit-known-issues

```diff
@@ -1,6 +1 @@
-https://npmjs.com/advisories/1316
-https://npmjs.com/advisories/1324
-https://npmjs.com/advisories/1084
-https://npmjs.com/advisories/1325
-https://npmjs.com/advisories/1179
```

Comment:
All these elements were already obsolete - hence cleaned up the file by occasion.